dist/angular-socialshare.js

```diff
@@ -514,6 +514,12 @@
           + ',top=' + ($window.innerHeight - attrs.socialsharePopupHeight) / 2 + ',left=' + ($window.innerWidth - attrs.socialsharePopupWidth) / 2);
       }
     }
+    , facebookMessengerShare = function facebookMessengerShare($window, $location, attrs, element) {
+
+      var href = 'fb-messenger://share?link=' + encodeURIComponent(attrs.socialshareUrl || $location.absUrl());
+
+      element.attr('href', href);
```

Comment:
Yes, just like `whatsapp` and `viber`, they can only work on a link that will trigger the app to open.